api/client/build.go

```diff
@@ -298,7 +292,14 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		out:         cli.out,
 		headers:     headers,
 	}
-	err = cli.stream("POST", fmt.Sprintf("/build?%s", v.Encode()), sopts)
+	var serveros string
+	serveros, err = cli.stream("POST", fmt.Sprintf("/build?%s", v.Encode()), sopts)
```

Comment:
Both fixed but now with consolidating the out parms, it's serverResp struct instead